Microsoft.Toolkit.Uwp.UI/Converters/OrientationToObjectConverter.cs

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.UI.Converters
+{
+    /// <summary>
+    /// This class returns a value depending on the <see cref="Orientation"/> of the value provided to the converter. In case of default will return the <see cref="VerticalValue"/>.
+    /// </summary>
+    public partial class OrientationToObjectConverter : DependencyObject, IValueConverter
+    {
+        /// <summary>
+        /// Identifies the <see cref="HorizontalValue"/> property.
+        /// </summary>
+        public static readonly DependencyProperty HorizontalValueProperty =
+            DependencyProperty.Register(nameof(HorizontalValue), typeof(object), typeof(TypeToObjectConverter), new PropertyMetadata(null));
+
+        /// <summary>
+        /// Identifies the <see cref="VerticalValue"/> property.
+        /// </summary>
+        public static readonly DependencyProperty VerticalValueProperty =
+            DependencyProperty.Register(nameof(VerticalValue), typeof(object), typeof(TypeToObjectConverter), new PropertyMetadata(null));
```

Comment:
Need to fix typeof() here referencing TypeToObjectConverter (found in Labs via crash on Skia (go Uno)). Fixed in labs.